ftw/slider/browser/slider.pt

```diff
@@ -19,10 +19,11 @@
                    tal:content="structure python:images.tag(alt='', title='')"
                    />
             <div class="sliderText"
-                 tal:condition="pane/text">
-              <p tal:condition="pane/title" tal:content="pane/title"
+                 tal:condition="python: pane.show_title or pane.text">
+              <p tal:condition="pane/show_title" tal:content="pane/title"
                   class="documentFirstHeading title"/>
-              <span tal:replace="structure pane/text/output" />
+              <span tal:condition="pane/text"
```

Comment:
Yes @maethu, I will prepare the PR in `ftw.sliderblock` once this PR has been merged and a release has been done because I will need to access the new `show_title` field from `ftw.slider`. `ftw.sliderblock` will then need at least the release containing these changes.